akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -146,6 +145,7 @@ private[remote] class Remoting(_system: ExtendedActorSystem, _provider: RemoteAc
           endpointManager = None
         }
 
+        import system.dispatcher
```

Comment:
As far as I can see, it is. endpointManager is an actor.
I have verified (with printlns) that the callback is running with the system.dispatcher in all cases when running the tests in akka-remote.